recipes/snzip/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set author = "kubo" %}
+{% set name = "snzip" %}
+{% set version = "1.0.4" %}
+{% set hash_type = "sha256" %}
+{% set hash_val = "a45081354715d48ed31899508ebed04a41d4b4a91dca37b79fc3b8ee0c02e25e" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.bz2
+  url: https://bintray.com/{{ author }}/generic/download_file?file_path={{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - snappy
+
+  run:
+    - snappy
```

Comment:
Forgot to ask. Do you know if the `snappy` version needs to match `snzip` version or not?